### PR TITLE
aur-repo: add --path

### LINKED ISSUE #555: aur-repo: alternative to command used in 1.5
I formerly used trizen -G (aurcheck -q -d revenge_repo) to fetch a list of my packages that needed updates and then automatically clone their aur repos. What would be the best alternative to this in the latest version of aurutils?

### PR DESCRIPTION
**Pull request by AladW**
Add the `--path` command-line option to print paths, and exit with `usage` if not specified. This way, the user must explicitely specify the operation they want to perform (issue #555).

Change `--repo-list` to only print the names of `file://` repositories, for easy processing where only these names are required (e.g. `aur-build -d` or `aur-sync -d`). The previous behavior, where paths are printed in full, can be achieved by combining `--repo-list` with `--path`.

----
In summary, we have the following options:

1. `aur repo --repo-list`
-> Print `file://` repository names
    
2. `aur repo --repo-list --path`
-> Print paths of all `file://` repositories
    
3. `aur repo --path`
-> Print path of single `file://` repository

4. `aur-repo`
-> Print usage text

----
Closes #555

### CODE REVIEW

**Comment by AladW** [comment]
I also added back `--repo` as alias to `--database` for all scripts, as it seems odd to have `aur repo --repo-list` but not `aur repo --repo`. This revisits issue #353.

**Comment by AladW** [comment]
Before merging, the documentation should be updated as well.

**Comment by AladW** [comment]
To keep some reasonable progress, let's keep track of documentation updates in #577.
